index.html

```diff
@@ -4,10 +4,165 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.0-alpha1/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-GLhlTQ8iRABdZLl6O3oVMWSktQOp6b7In1Zl3/Jr59b6EGGoI1aFkw7cmDA6j6gD" crossorigin="anonymous">
-  <title>Document</title>
+  <link rel="stylesheet" href="./assets/css/bootstrap-mod.css">
+  <title>Home</title>
 </head>
 <body>
-  
+  <header class="p-3 bg-hero" id="navbar">
+    <nav class="navbar navbar-expand-lg navbar-light p-3 mx-lg-5">
+      <div class="container-fluid">
+        <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarScroll" aria-controls="navbarScroll" aria-expanded="false" aria-label="Toggle navigation">
+          <span class="navbar-toggler-icon"></span>
+        </button>
+        <div class="collapse navbar-collapse justify-content-lg-between" id="navbarScroll">
+          <div class="navbar-brand">
+            <img class="generic-logo" src="./assets/img/generic-logo.svg" alt="">
+          </div>
+          <ul class="navbar-nav my-2 my-lg-0 navbar-nav-scroll" style="--bs-scroll-height: 100px;">
+            <li class="nav-item">
+              <a class="nav-link active" aria-current="page" href="./index.html">Home</a>
+            </li>
+            <li class="nav-item">
+              <a class="nav-link" href="./about.html">About</a>
+            </li>
```

Comment:
- [ ] When I clicked on `About link` for the mobile version, I noticed these `nav` items showing up; kindly hide it. 👍 

![download__8_-removebg-preview](https://user-images.githubusercontent.com/103386183/210819852-e2d979ad-de53-4af3-9136-c06b5f175c1d.png)